### DISCUSSION TITLE
Development setup [Jul 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2814] [Discord Threads]
> 🙋‍♂️ **Alsu**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@955863278133870672> Please confirm if you are following the steps listed [here](https://docs-dev.nocodb.com/engineering/development-setup/)  
#  
> 🙋‍♂️ **Alsu**  
#### I am running it with docker docker run -d --name nocodb-mysql \   
#### -v "$(pwd)"/nocodb-develop:/usr/app/data/ \   
#### -p 80:8080 \ nocodb/nocodb:latest  
#  
> 👨‍🦲 **dstala**  
#### making changes locally will not update docker image. you will have to switch over to node based `Development setup` for applying your modifications  
#  
> 🙋‍♂️ **Alsu**  
#### Thank you!  
#  
> 🙋‍♂️ **Alsu**  
#### I went through instructions to launch with node, I made small changes in signin.vue file, relaunch it,  front end seemed to pick up changes (in my terminal files were reloaded), but nothing changes in the browser  
#  
> 🙋‍♂️ **Alsu**  
#### Do you know what else I need to do to be able see changes?  
#  
> 🙋‍♂️ **Alsu**  
#### Never mind, changes are in port 3000, I was checking 8080  
#  
> 👨‍🦲 **dstala**  
#### <@955863278133870672>, do mark your feedback here   
#### https://discord.com/channels/661905455894888490/953304597722718218  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/996813100827283546)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
